doc/source/drivers/raster/stacit.rst

```diff
@@ -108,17 +112,47 @@ Examples
 List the subdatasets associated to a `STAC search <https://github.com/radiantearth/stac-api-spec/tree/master/item-search>`_
 on a given collection, bbox and starting from a datetime:
 
-::
+.. tabs::
+
+   .. code-tab:: bash
 
     gdalinfo "STACIT:\"https://planetarycomputer.microsoft.com/api/stac/v1/search?collections=naip&bbox=-100,40,-99,41&datetime=2019-01-01T00:00:00Z%2F..\""
+    # from GDAL 3.11+
+    gdal raster info "STACIT:\"https://planetarycomputer.microsoft.com/api/stac/v1/search?collections=naip&bbox=-100,40,-99,41&datetime=2019-01-01T00:00:00Z%2F..\""
+
+
+   .. code-tab:: ps1
+
+    gdalinfo 'STACIT:\"https://planetarycomputer.microsoft.com/api/stac/v1/search?collections=naip&bbox=-100,40,-99,41&datetime=2019-01-01T00:00:00Z%2F..\"'
+    # from GDAL 3.11+
+    gdal raster info 'STACIT:\"https://planetarycomputer.microsoft.com/api/stac/v1/search?collections=naip&bbox=-100,40,-99,41&datetime=2019-01-01T00:00:00Z%2F..\"'
+
+   .. tab:: Python
 
+      .. literalinclude :: examples/stacit.py
+         :language: python
+         :end-before: # subdatasets
```

Comment:
```suggestion
         :start-after: # Example 1
         :end-before: # Example 2
```

A convention like this would make the `stacit.py` file more clear, I think.

Reply:
Restructured as suggested.